shell/platform/fuchsia/dart-pkg/zircon/lib/src/system.dart

```diff
@@ -236,7 +236,9 @@ class System extends NativeFieldWrapperClass2 {
   static MapResult vmoMap(Handle vmo) native 'System_VmoMap';
 
   // Time operations.
-  static int clockGetMonotonic() native 'System_ClockGetMonotonic';
+  static int clockGetMonotonic() {
```

Comment:
btw... I don't think we currently have dart unittest/integration test coverage for this method: https://cs.opensource.google/search?q=%22clockGetMonotonic%22&sq=&ss=fuchsia%2Ffuchsia:sdk%2Fdart%2Fzircon%2Ftest%2F

Just wanted to make sure if you've manually checked if this worked since our existing tests don't cover this. 🙂

Reply:
I noticed it, I was testing using https://fuchsia-review.googlesource.com/c/fuchsia/+/554605